tests/Feature/LMSJoinClassTest.php

```diff
@@ -18,88 +18,87 @@ public function testJoinClassWithoutLoginExpectResponseStatus401()
     $response->assertStatus(401);
   }
 
-  public function testJoinClassAsAuthorizedUserWithUnknownClassCodeExpectResponseStatus500()
+  public function testJoinClassAsAuthorizedUserWithUnknownClassCodeExpectResponseStatus404()
   {
-      $user = User::factory()->create();
-      $user->rawPassword = 'password';
+    $user = User::factory()->create();
+    $user->rawPassword = 'password';
 
-      $url = route('badaso.course.join');
+    $url = route('badaso.course.join');
 
-      $response = AuthHelper::asUser($this, $user)->json('POST', $url, [
-          'code' => 'xxx'
-      ]);
-      $response->assertStatus(500);
+    $response = AuthHelper::asUser($this, $user)->json('POST', $url, [
+      'code' => 'xxx'
+    ]);
+    $response->assertStatus(404);
```

Comment:
backend code should be indented with 4 spaces

Reply:
alright got it